README.md

```diff
@@ -4,25 +4,8 @@ WebLogic Monitoring Exporter
 [![Build Status](https://travis-ci.org/oracle/weblogic-monitoring-exporter.svg?branch=master)](https://travis-ci.org/oracle/weblogic-monitoring-exporter)
 
 The WebLogic Monitoring Exporter uses the [WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111) to scrape runtime information and then exports [Prometheus](http://prometheus.io)-compatible metrics.
-It is deployed as a web application in a WebLogic Server (WLS) instance, version 12.2.1 or later, typically, in the instance from which you want to get metrics.
-
-## Downloading the release
-
-You can find all the releases on the [Releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).
-
-To download the web application `wls-exporter.war` and put your configuration file into the WAR, download the `getXXX.sh` script, which is provided with each release and also can be downloaded from the Releases page, and then run:
-
-```
-bash getXXX.sh <your-config-file>
-```
-
-## Building from source
-
-Use `mvn install` to build the web application. This will create `wls-exporter-<version>`, where _version_
-is the Maven-assigned version number.
-
-Adding `-Dconfiguration=<some-config-file>` will insert the specified configuration as its default and remove
-the version number to simplify deployment to WebLogic Server.
+It is available both as a [web application](#web-application) to be deployed to the WebLogic Server (WLS) instance, version 12.2.1 or later, from which you want to get metrics, 
+and also as a [sidecar](#sidecar), which runs as its own process, and which is configured to contact a WLS instance.
 
 ## Configuration
 Here is an example `yaml` configuration:
```

Comment:
example `yaml` configuration -> example `yaml` file configuration (old text; I should have corrected it previously)